src/components/CommentInput/CommentInput.styled.js

```diff
@@ -61,16 +61,32 @@ export const Comment = styled.div`
 
   /* check box styles */
   .ql-editor ul {
+    /* checkbox data-checked false or true */
+    &[data-checked] {
+      li {
+        height: 25px;
+        /* tick circle */
+        &::before {
+          margin-right: 12px;
+          font-size: unset;
+          top: -2px;
+          position: relative;
+        }
+
+        /* cursor too height */
```

Comment:
I feel this may be wrong in grammar?
```suggestion
        /* fix cursor too high */
```
Or
```suggestion
        /* cursor to height */
```

Reply:
Thanks, I guess i'll start baking my cake of shame...